src/components/Layout/MainLayout.jsx

```
@@ -1159,21 +1160,90 @@ const MainLayout = (props) => {
+                              (pageName === "m5" || pageName === "wordsorimage")
+                                ? "25px"
+                                : "50px"
+                            }
+                            width={
+                              isMobile &&
+                              (pageName === "m5" || pageName === "wordsorimage")
+                                ? "25px"
+                                : "50px"
+                            }
+                            style={{ flexShrink: 0 }}
+                          />
```

ESLint

[error] Missing "key" prop for element in iterator

(react/jsx-key)